.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# See the code-owners Github docs:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# Mostly backend-y directories
+/bedrock @mozilla/bedrock-codeowners-backend
```

Comment:
**suggestion:** Maybe we lose this one as it'll spam the backend team with every single change

Reply:
Higher up is more a default, so anything below that is also in `/bedrock` will take precedence. E.g. if we add the `*.html` rule below for FE, that will override this.